internal/connection_repository/sql_connection_locator.go

```diff
@@ -134,21 +134,28 @@ func (scm *SqlConnectionLocator) GetConnectionsByAccount(ctx context.Context, ac
 
 	for rows.Next() {
 		var client_id domain.ClientID
+		var orgIdString string
 		var dispatchersString string
-		if err := rows.Scan(&client_id, &dispatchersString, &totalConnections); err != nil {
+		if err := rows.Scan(&client_id, &orgIdString, &dispatchersString, &totalConnections); err != nil {
 			logger.LogError("SQL scan failed.  Skipping row.", err)
 			continue
 		}
 
+		var org_id domain.OrgID
+		err = json.Unmarshal([]byte(orgIdString), &org_id)
```

Comment:
The org_id will be stored as a string in the database.  So instead of doing a `json.Unmarshal()`, you'll just need to assign `orgIdString.String`  to  `orgId`.